public/static/docs/tutorials/pipelines.md

```diff
@@ -51,7 +51,7 @@ $ git commit -m "Download and add code to new Git repo"
 > `dvc get` can use any <abbr>DVC project</abbr> hosted on a Git repository to
 > find the appropriate [remote storage](/doc/command-reference/remote) and
 > download <abbr>data artifacts</abbr> from it. (It works like `wget`, but for
-> DVC repositories.) In this case we use
+> Git repositories.) In this case we use
```

Comment:
OK, this one makes more sense to change given the context, thanks... BUT actually to keep the tutorial simple I would leave this as it was, or use the following for emphasis (we care about the repo being a DVC project in this context):
```suggestion
> DVC or Git repositories.) In this case we use
```

Reply:
Yeah, makes sense. If you're in the market for a git downloader you're probably not going to find DVC.

Reply:
Yep. Thanks